<launch>
    
    <arg name="debug" default="0"/>

    <rosparam file="$(find ebobot)/config/nodes/costmap_server.yaml" />
    

    <node pkg="ebobot" type="costmap_server.py" name="costmap_server" output="log" respawn="true" respawn_delay="10"> 
        <param name = "debug" value = "$(arg debug)"/>
        <param name = "file_dir" value = "$(find ebobot)/config/costmap"/>
        <param name = "file" value = "$(find ebobot)/config/costmap/costmap.png"/>
    </node>
    
</launch>